includes/Integrations/Core_Themes_Support.php

```diff
@@ -77,7 +87,27 @@ public function extend_theme_support() {
 	 */
 	public function embed_web_stories() {
 		$customizer = new Customizer();
-		echo $customizer->render_stories(); // phpcs:ignore -- WordPress.Security.EscapeOutput.OutputNotEscaped - Escaped web stories HTML.
+		$this->enqueue_style( $this->style_handle, [ Renderer::STYLE_HANDLE ] );
```

Comment:
```suggestion
		$this->enqueue_style( 'web-stories-theme-style-' . get_stylesheet();, [ Renderer::STYLE_HANDLE ] );
```

Let's get rid of the `$style_handle` class property